nixos/modules/hardware/cpu/amd-ryzen-smu.nix

```
@@ -0,0 +1,26 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.hardware.cpu.amd.ryzen_smu;
+  ryzen_smu = config.boot.kernelPackages.ryzen_smu;
+in
+{
+  options.hardware.cpu.amd.ryzen_smu = {
+    enable = mkEnableOption (lib.mdDoc ''
```

```suggestion
    enable = mkEnableOption ''
```
`lib.mdDoc` is now just an alias and can be safely removed everywhere.
see https://github.com/NixOS/nixpkgs/commit/d36f950d405369abca5144b14a201bc25c4a080a and https://github.com/NixOS/nixpkgs/pull/237557